app/views/documents_v2/autolaunch.html.haml

```diff
@@ -0,0 +1,72 @@
+-# If the requested document was found then this page waits for Lara to tell it the interactive run state
+-# via iFramePhone.  If there are no access keys in the interactive run state then a call is made back into the
+-# document store to copy the shared document and then a keys are then sent back via posted back
```

Comment:
Is this true that it makes a copy in the document store? I thought that the CFM made the copy after CODAP was launched.